playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
```

Comment:
And one other note, we have users running playbooks with using `--tags`.  I've seen bugs related to it not working properly so it definitely needs to be addressed.